lib/cloud_controller/deployment_updater/updater.rb

```diff
@@ -22,7 +22,18 @@ def scale
 
       def canary
         with_error_logging('error-canarying-deployment') do
-          Actions::Canary.new(deployment, logger).call
+          # TODO: do we need to pass in deployment.original_web_process_instance_count + 1 if there is a single canary instance?
```

Comment:
we can remove this, this was resolved but taking `deployment.canary_total_instances`